lib/modules/hosts/superstack.js

```diff
@@ -0,0 +1,42 @@
+import { ajax } from '../../environment';
+
+const IMAGE = 'IMAGE';
+const GALLERY = 'GALLERY';
+
+const superstack = {
```

Comment:
This can be: `export default {`; the `superstack` var isn't used anywhere besides the export